docs/pages/en/4.providers/imagekit.md

```diff
@@ -18,3 +18,271 @@ export default {
   }
 }
 ```
+
+## ImageKit `fit` Parameters
+
+In addition to the standard [`fit`](/components/nuxt-img#fit) properties of Nuxt Image and Nuxt Picture, ImageKit offers more cropping and resizing options to the users:
+
+* `extract` - The output image has its height, width as requested and the aspect ratio is preserved. Unlike `cover` parameter, here we extract out a region of the requested dimension from the original image.
+* `pad_extract` - This parameter is similar to `extract`. This comes in handy in scenarios where we want to extract an image of larger dimension from a smaller image. So, the `pad_extract` mode adds a solid colored padding around the image to make it match the exact size requested.
+
+Read ImageKit [crop and crop mode documentation](https://docs.imagekit.io/features/image-transformations/resize-crop-and-other-transformations#crop-crop-modes-and-focus) for more details and examples of how it works.
+
+## ImageKit Modifiers
+
+On top of the standard [Nuxt Image modifiers](/components/nuxt-img#modifiers), a user can also leverage ImageKit-specific transformation parameters provided in the `modifier` prop.
+
+### `focus`
+
+This parameter can be used along with resizing and cropping to focus on the desired part of the image. You can use focus parameter values like `left`, `right`, `top`, `bottom`, `center`, `top`, `left`, `bottom`, `right`, `top_left`, `top_right`, `bottom_left` and `bottom_right`.
+
+Custom coordinates can also be used to focus using parameter value `custom`.
+
+Moreover, ImageKit also provides smart cropping that can automatically detect most important part of the image using `auto`. And, `face` can be used to find out a face (or multiple faces) in an image and focus around that.
+
+See below example of how face cropping can be done in Nuxt Image using ImageKit-
+
+```html
+<nuxt-img
+  provider="imagekit"
+  src="/img/girl.jpeg"
+  width="150"
+  height="150"
+  :modifiers="{ focus: 'face' }"
+/>
+```
+
+Note: Set `your_imagekit_id` to `demo` in `nuxt.config.js` to run the above demo. 
+
+Check out ImageKit's documentation on [focus](https://docs.imagekit.io/features/image-transformations/resize-crop-and-other-transformations#focus-fo) to learn more.
+
+
+### `blur`
+
+This can be used to blur an image. Use modifier `blur` to specify the Gaussian Blur radius that is to be applied on the image. Possible values include integers between `1` to `100`.
+
+```html
+<nuxt-img
+  provider="imagekit"
+  src="/medium_cafe_B1iTdD0C.jpg"
+  :modifiers="{ blur : 10}"
+/>
+```
+
+Note: Again here, set `your_imagekit_id` to `demo` in `nuxt.config.js` to run this example. These examples can also be found in the ImageKit documentation. You can use this and many more powerful transformations on your own images with your ImageKit ID!
```

Comment:
I can understand that they are meant to be references. But what I have done makes it an example as a well as a reference guide. In our SDK’s too we provide sample apps. I feel it would be even more easier for developer to understand through examples with instructions to get a demo quickly (cloudinary does not have a guide to use their demo and hence I spent some extra time figuring out how to run and test). Moreover, apart from us only 2 providers have provided reference, cloudinary and storyblok, and they also have provided working examples, not something like default-image.

Reply:
With Niksh on this one. Let's follow the same convention as other SDKs and docs.

`/default-image.jpeg` is an image we upload during account setup. All our SDKs use this. If the user enters the endpoint once then, it just works unless someone has explicitly deleted the default-image.jpeg (a small number of users, I would not worry about it).

Also, avoid stating the setup instruction in this section. How to configure endpoint etc., should be at the top and not repeated in any sub-section.

Reply:
Incorporated the feedback and made changes.